paket.lock

```diff
@@ -261,83 +238,20 @@ NUGET
       System.Threading.Overlapped (>= 4.3)
       System.Threading.Tasks (>= 4.3)
       System.Threading.Thread (>= 4.3)
-    System.Linq (4.3)
-      System.Collections (>= 4.3)
-      System.Diagnostics.Debug (>= 4.3)
-      System.Resources.ResourceManager (>= 4.3)
-      System.Runtime (>= 4.3)
-      System.Runtime.Extensions (>= 4.3)
-    System.Linq.Expressions (4.3)
-      System.Collections (>= 4.3)
-      System.Diagnostics.Debug (>= 4.3)
-      System.Globalization (>= 4.3)
-      System.IO (>= 4.3)
-      System.Linq (>= 4.3)
-      System.ObjectModel (>= 4.3)
-      System.Reflection (>= 4.3)
-      System.Reflection.Emit (>= 4.3)
-      System.Reflection.Emit.ILGeneration (>= 4.3)
-      System.Reflection.Emit.Lightweight (>= 4.3)
-      System.Reflection.Extensions (>= 4.3)
-      System.Reflection.Primitives (>= 4.3)
-      System.Reflection.TypeExtensions (>= 4.3)
-      System.Resources.ResourceManager (>= 4.3)
-      System.Runtime (>= 4.3)
-      System.Runtime.Extensions (>= 4.3)
-      System.Threading (>= 4.3)
-    System.ObjectModel (4.3)
-      System.Collections (>= 4.3)
-      System.Diagnostics.Debug (>= 4.3)
-      System.Resources.ResourceManager (>= 4.3)
-      System.Runtime (>= 4.3)
-      System.Threading (>= 4.3)
-    System.Reactive (3.1.1)
-      System.Reactive.PlatformServices (>= 3.1.1)
-    System.Reactive.Core (3.1.1)
-      System.ComponentModel (>= 4.0.1)
-      System.Diagnostics.Contracts (>= 4.0.1)
-      System.Dynamic.Runtime (>= 4.0.11)
-      System.Reactive.Interfaces (>= 3.1.1)
-    System.Reactive.Interfaces (3.1.1)
-      NETStandard.Library (>= 1.6)
-    System.Reactive.Linq (3.1.1)
-      System.Reactive.Core (>= 3.1.1)
-      System.Runtime.InteropServices.WindowsRuntime (>= 4.0.1)
-    System.Reactive.PlatformServices (3.1.1)
-      System.Reactive.Linq (>= 3.1.1)
+    System.Reactive (4.0)
```

Comment:
Ok this update broke it :)

Reply:
Ok, so we couln'd avoid it because we needed this update for the test :)

Thanks for pointed it out.

Reply:
Thanks for the fix and all the solutions provided :)

I tested rc17 and it's indeed working :clap: